app/assets/javascripts/ohm.style.js

```diff
@@ -1,6 +1,6 @@
 // enable RTL support e.g. Arabic & Hebrew text
-mapboxgl.setRTLTextPlugin(
-  'https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-rtl-text/v0.2.3/mapbox-gl-rtl-text.js',
+maplibregl.setRTLTextPlugin(
+  'https://unpkg.com/@mapbox/mapbox-gl-rtl-text@0.2.3/mapbox-gl-rtl-text.min.js',
```

Comment:
We should build the package ourselves and serve it up as part of this site rather than relying on yet another CDN. (This can be tail work.)